src/renderers/controls/slider.control.tsx

```diff
@@ -0,0 +1,61 @@
+import { JSX } from '../JSX';
+import * as _ from 'lodash';
+import {
+  and,
+  optionIs,
+  RankedTester,
+  rankWith,
+  schemaTypeIs,
+  uiTypeIs
+} from '../../core/testers';
+import { Control, ControlProps, ControlState } from './Control';
+import {
+    formatErrorMessage,
+    mapStateToControlProps,
+    registerStartupRenderer
+} from '../renderer.util';
+import { connect, Event } from '../../common/binding';
+
+export const sliderControlTester: RankedTester = rankWith(4, and(
+    uiTypeIs('Control'),
+    schemaTypeIs('number'),
```

Comment:
This might also be applicable for integers, doesn't it? Also, one might argue, but I'd only use a slider if there is a `minimum` and `maximum` configured (and you already do in the example) since sensible default rarely can be provided.

Reply:
do we have an `or`? I agree that we should also support integers.

Reply:
We don't have an `or` right now. I think we need a different tester to check this

Reply:
I created another function called `or` which is basically similar to `and`. I changed the logical expression `testers.reduce((acc, tester) => acc || tester(uischema, schema), true);` . Is it ok?

Reply:
Yes should be fine if you place it within `testers.ts`.

Reply:
should the initial value be `false`?

Reply:
Uhm, yes, missed that.

Reply:
I see, the reason is if the initial value start with true, even tester returns false (neither number nor integer), the result of `or` will be always true which is not correct. Am I right?